webapp-django/crashstats/crashstats/models.py

```diff
@@ -210,7 +210,10 @@ def collected_date(self):
         return "20" + self.crash_id[-6:]
 
     def report_url(self):
-        return reverse("crashstats:report_index", args=(self.crash_id,))
+        try:
+            return reverse("crashstats:report_index", args=(self.crash_id,))
+        except NoReverseMatch:
+            return f"no reverse match: {self.crash_id}"
```

Comment:
Adding this lets me be stupid and create MissingProcessCrash records for crash ids that are invalid.